### ISSUE TITLE
Recent update breaks flake's module

### ISSUE THREAD

**Issue by PaulGrandperrin** [Mic92/nix-ld#35]
After updating from `7d251c0c5adf6b9b003499243be257d0f130b3d6` to `3210f71f0171b491b88bbb099effc5d3c7ef99e9`, I get this error:

```
error: The option `programs.nix-ld.package' does not exist. Definition values:
       - In `/nix/store/fcr15jsj9nrbpzsm0c4hpvd9gd9w8ay0-source/flake.nix': <derivation /nix/store/3x7p9ds58969q2cgvnfmpkmcz2wvcvad-nix-ld.drv>
```

I'm importing the flake module `inputs.nix-ld.nixosModules.nix-ld` and then setting `programs.nix-ld.enable = true;`.

**Comment by atkrad** [comment]
I have same issue.

**Comment by Mic92** [comment]
Fixed again. This option only exists in nixpkgs-unstable. You might be also able to just drop the flake and use the nix-ld option in 22.11 unless you want to test changes only available in the repository.

**Comment by PaulGrandperrin** [comment]
I didn't realize `nix-ld` was available in `21.11`!

Thanks for you work!
